Client/src/Components/UserReview/DatasetModal.tsx

```diff
@@ -1,91 +1,85 @@
-import { Box, Grid, Paper, Typography } from '@material-ui/core'
-import React, { useEffect, useRef, useState } from 'react'
+import { Dialog, DialogActions, DialogContent, DialogTitle, Grid, Typography } from '@material-ui/core'
+import { FormikProps, FormikValues } from 'formik'
+import React, { useRef, useState } from 'react'
 
 import CancelIcon from "@material-ui/icons/Cancel"
 import { DatasetForm } from '../DatasetUpload/DatasetForm'
-import { FormikProps } from 'formik'
+import { DefaultFormFooter } from '../Forms/DefaultFormFooter'
 import { IApprovedDatasetModel } from '../../Models/Datasets/IApprovedDatasetModel'
 import { IDatasetModel } from '../../Models/Datasets/IDatasetModel'
-import Modal from '@material-ui/core/Modal/Modal'
 import { ModalFooter } from './ModalFooter'
-import { ModalFooterApprove } from './ModalFooterApprove'
-import { classStyles } from '../../appTheme'
-import { fixPartialForform } from '../DatasetUpload/DatasetView'
 
 interface IProps {
-  singleDataset: IApprovedDatasetModel,
+  open: boolean
+  dataset: IApprovedDatasetModel,
+  setOpen: (value: boolean) => void
   handleApproveDataset: (datasetId: number) => void,
   handleDeleteDataset: () => void,
   handleSubmitDataset: (formDataset: IDatasetModel) => void,
-  setOpen: (value: boolean) => void
-  open: boolean
 }
 
 export const DatasetModal = (props: IProps) => {
-
-  const { open, setOpen } = { ...props }
-  const [dataset, setDataset] = useState<IDatasetModel>()
-  const [editable, setEditable] = useState(false)
-  const mappedDataset: IDatasetModel = { ...props.singleDataset }
-  const { handleApproveDataset, handleDeleteDataset, handleSubmitDataset } = { ...props }
-  const [editOn, steEditOn] = useState(true)
+  const { open, dataset, setOpen, handleApproveDataset, handleDeleteDataset, handleSubmitDataset } = { ...props }
+  const [isEditMode, setEditMode] = useState(false)
   const formikReference = useRef<FormikProps<unknown>>()
 
-  useEffect(() => {
-    setDataset(fixPartialForform(mappedDataset))
```

Comment:
this was unnecessary, the datasetForm fixes input itself when a dataset is passed